src/components/Navbar/index.tsx

```diff
@@ -28,9 +23,8 @@ const IconBox = styled(Box)<BoxProps & DisplayProps>`
 	cursor: pointer;
 `;
 
-const MenuBox = styled(Flex)<FlexProps & MaxHeightProps & DisplayProps>`
-	${display}
-	${maxHeight};
+const MenuBox = styled(Flex)<FlexProps & DisplayProps>`
+	${display};
```

Comment:
While we are doing this, let's add `display` as well to `asRendition`, as it is quite useful for responsiveness.